src/app/containers/create-movie/formField.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+const FormField = ({ id, label, formik }) => (
+  <div className="formGroup">
+    <label className='formLabel' htmlFor={id}>{label}</label>
+    <input
```

Comment:
The Field component from formik also handles errors inside. It's a good practice not to reinvent the wheel.

Reply:
A bit of a larger commit for this but i think it's an improvement, see [40cbf22](https://github.com/OctaFloare/flitnext-2024/pull/12/commits/40cbf22585314c0365ec2f1e9adc44891df1f6f9)  and [846c45f](https://github.com/OctaFloare/flitnext-2024/pull/12/commits/846c45f6ee3f0824fe6be6805f3e90199466f0b7) for the field